### PR TITLE
Add schedule details in focus card

### LINKED ISSUE #195: Add `interview` details into `focusCard`

### PR DESCRIPTION
**Pull request by LeeEnEn**
Focus card when there is no interview for the candidate:
image

When there is an interview:
image

Closes #195

### CODE REVIEW

**Comment by domlimm** [comment]
wow it rocks! ❤️ ______ ❤️

File: src/main/java/seedu/address/ui/FocusCard.java [hunk 1]
```diff
@@ -205,4 +219,18 @@ private void setProfilePicture(Name candidateName) {
...
+
+    private void setSchedule() {
+        if (interview != null) {
+            DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd LLLL yyyy");
+            String temp = interview.getInterviewDate().getDayOfWeek().toString();
+            String d = temp.charAt(0) + temp.substring(1).toLowerCase() + ",";
+            day.setText(d);
+            date.setText(interview.getInterviewDate().format(formatter));
+            time.setText("@ " + interview.getInterviewStartTime().toString());
+            scheduleMessage.setText(SCHEDULE_MESSAGE);
```
Comment (domlimm) [inline src/main/java/seedu/address/ui/FocusCard.java:231]: En En nice job! Only comment is I actually thought that it's 1 `Label` for all 3 fields. But no issue at all! Darn good job!

**Review by domlimm — APPROVED** [review]
No issues at all, LGTM! Good job @LeeEnEn! 💯

**Review by tiewweijian — APPROVED** [review]
LGTM ! I think we can pull up our Codecov along the way with other test cases :)

**Pull request merged**